docs/topics/multiplatform/multiplatform-run-tests.md

```diff
@@ -1,67 +1,375 @@
-[//]: # (title: Run tests with Kotlin Multiplatform)
+[//]: # (title: Test your multiplatform app − tutorial)
 
-By default, Kotlin supports running tests for JVM, JS, Android, Linux, Windows, macOS as well as iOS, watchOS, and tvOS simulators. 
-To run tests for other Kotlin/Native targets, you need to configure them manually in an appropriate environment, emulator, 
-or test framework.
+In this tutorial, you'll learn how to create, configure, and run tests in Kotlin Multiplatform applications.
 
-## Required dependencies
+Tests for multiplatform projects can be divided into two categories:
 
-The [`kotlin.test` API](https://kotlinlang.org/api/latest/kotlin.test/) is available for multiplatform tests. When you
-[create a multiplatform project](multiplatform-library.md), the Project Wizard automatically adds test dependencies to
-common and platform-specific source sets. 
+* Tests for common code. These tests can be run on any platform using any supported framework.
+* Tests for platform-specific code. These are essential to test platform-specific logic. They use a platform-specific
+  framework and can benefit from its additional features, such as a richer API and a wider range of assertions.
 
-If you didn't use the Project Wizard to create your project, you can [add the dependencies manually](gradle-configure-project.md#set-dependencies-on-test-libraries).
+Both categories are supported in multiplatform projects. This tutorial will show you how to set up, create, and run unit
+tests in a simple Multiplatform project. Then you'll work with a more complex example that requires tests both for common
+and platform-specific code.
 
-## Run tests for one or more targets
+> This tutorial assumes that you are familiar with:
+> * The layout of a Kotlin Multiplatform project. If this is not the case,
+    complete [this tutorial](multiplatform-mobile-getting-started.md) to get started.
+> * The basics of popular unit testing frameworks, such as [JUnit](https://junit.org/junit5/).
+>
+{type="tip"}
 
-To run tests for all targets, run the `check` task.
+## Test a simple Multiplatform project
```

Comment:
Should multiplatform be lowercase? It isn't in a [similar heading](https://kotlinlang.org/docs/multiplatform-mobile-ktor-sqldelight.html#create-a-multiplatform-project) in the Ktor tutorial but I'm not sure if that is a mistake 🤔 The Rules heading later in the doc also has it in lowercase.

Reply:
It's capitalized when we underline that it's a Kotlin Multiplatform project, mentioning the technology
Here I agree it's better to make it consistent with the Rules header, and update the Ktor tutorial later